nixos/modules/services/web-apps/kasmweb/default.nix

```diff
@@ -124,29 +124,20 @@ in
   };
 
   config = lib.mkIf cfg.enable {
-
     systemd.services = {
       "init-kasmweb" = {
         wantedBy = [
           "docker-kasm_db.service"
+          "podman-kasm_db.service"
         ];
-        before = [
-          "docker-kasm_db.service"
-          "docker-kasm_redis.service"
-          "docker-kasm_db_init.service"
-          "docker-kasm_api.service"
-          "docker-kasm_agent.service"
-          "docker-kasm_manager.service"
-          "docker-kasm_share.service"
-          "docker-kasm_guac.service"
-          "docker-kasm_proxy.service"
-        ];
+        wants = ["network-online.target"];
+        after = ["network-online.target"];
         serviceConfig = {
           Type = "oneshot";
           ExecStart = pkgs.substituteAll {
             src = ./initialize_kasmweb.sh;
             isExecutable = true;
-            binPath = lib.makeBinPath [ pkgs.docker pkgs.openssl pkgs.gnused ];
+            binPath = lib.makeBinPath [ pkgs.docker pkgs.openssl pkgs.gnused pkgs.yq-go ];
```

Comment:
Sorry, I'm not an expert. Does this specifically depend on docker? is there a way to depend on either docker or the podman compatibility docker command?